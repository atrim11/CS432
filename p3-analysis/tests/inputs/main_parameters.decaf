def int main(int argc, bool argv) {
    return -1;
}