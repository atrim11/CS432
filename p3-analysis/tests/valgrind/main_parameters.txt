==815393== Memcheck, a memory error detector
==815393== Copyright (C) 2002-2022, and GNU GPL'd, by Julian Seward et al.
==815393== Using Valgrind-3.22.0 and LibVEX; rerun with -h for copyright info
==815393== Command: ../decaf inputs/main_parameters.decaf
==815393== 
Main function on line 1 must have no parameters
==815393== 
==815393== HEAP SUMMARY:
==815393==     in use at exit: 0 bytes in 0 blocks
==815393==   total heap usage: 1,781 allocs, 1,781 frees, 212,412 bytes allocated
==815393== 
==815393== All heap blocks were freed -- no leaks are possible
==815393== 
==815393== For lists of detected and suppressed errors, rerun with: -s
==815393== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
