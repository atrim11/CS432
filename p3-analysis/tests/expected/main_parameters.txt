Main function on line 1 must have no parameters
